chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/src/org/eclipse/chemclipse/msd/converter/supplier/mzdata/converter/ChromatogramImportConverter.java

```diff
@@ -8,35 +8,65 @@
  * 
```

Comment:
Adjust the year to 2021.
Add yourself to the committer list if relevant parts of the code have been edited by yourself.

Reply:
This is just copy and pasted code. Nothing is from me or 2021.

Reply:
The change (copying) has been made in 2021, so the header needs to be updated.